.vscode/settings.json

```diff
@@ -30,7 +30,7 @@
   "python.linting.pylintCategorySeverity.refactor": "Information",
   "python.linting.pylintEnabled": true,
   "python.linting.pylintUseMinimalCheckers": false,
-  "python.testing.pytestArgs": ["test", "--no-cov"],
+  "python.testing.pytestArgs": ["test/unit", "--no-cov"],
```

Comment:
I would prefer to have them all in there, you can still select the group of tests to run. Or whats your reasoning behind this?

Reply:
Mainly so that you can run all tests at once. Unfortunately, VSCode cannot yet run specific folders.

Reply:
We just need some easy thing to also be able to debug non unit tests

Reply:
5ca7388fefe03b4e7804b6aabc4e65dc4bf7aa34
Only thing that bothers me is that it's not possible anymore to run only the tests in the `unit` folder. But that's a shortcoming of VSCode.
![image](https://user-images.githubusercontent.com/29308176/90264363-489a2400-de51-11ea-877e-f822141cba6d.png)

Reply:
yeah

Reply:
https://github.com/microsoft/vscode-python/issues/7188 looks like this wont come any time soon. 
I think we just have to get more unit tests and be able to get rid of those channel tests...

Reply:
https://github.com/ComPWA/expertsystem/blob/1d682f52b530d1a75a62a189e642f3ed0261847f/tests/unit/amplitude/test_yaml_helicity.py#L146-L148
Seems that the test is just incomplete. But now I remember this test sort of stalled while #22 is in the making. We didn't want to put too much effort into this temporary `_yaml_adapter` thingy until then.

Reply:
For me it would be enough if I can debug those channel files as a file, so plugging in `if __name__ == "__main__":`
Then we can remove those channel tests from vscode and all is good

Reply:
Yes, I think it would be cool to have the (or some) channel tests as notebooks.

Reply:
> For me it would be enough if I can debug those channel files as a file, so plugging in if __name__ == "__main__":

Oh interesting, seems that they already have that. Only not the nbody tests, because they make use of `pytest.mark.parametrize`...

Reply:
I guess I would just leave all tests in there then. And if you want to run only unit tests you could use the terminal right. That would be an ok tradeoff for now

Reply:
Personally I want to "run all tests" with a quick shortkey in VSCode without having to open a terminal for that.